.github/workflows/book.yml

```diff
@@ -0,0 +1,32 @@
+name: halo2 book
+
+on:
+  push:
+    branches:
+      - master
+      - next
+
+jobs:
+  deploy:
+    runs-on: ubuntu-18.04
```

Comment:
Could we replace this with `ubuntu-latest`?

Reply:
fixed